src/lib/adapter.ts

```diff
@@ -148,6 +149,23 @@ class GleeAdapter extends EventEmitter {
     return this._serverUrlExpanded
   }
 
+  async resolveConfig(protocol: string) {
+    if(!this.glee.options[protocol]) return undefined
+    const protocolConfig = {...this.glee.options[protocol]}
```

Comment:
Why are you making a copy of the object? 🤔

Reply:
I am making a copy of the original config object so that I still have the unresolved functions. Any adapter can access it by doing `this.glee.options`. I did this because If any adapter wants to resolve the config functions every time a message is received, it can do so. (I don't know if this case will ever happen)

Reply:
Oh, alright 👍 Sounds good!